ChecksFinderClient.py

```diff
@@ -56,34 +68,33 @@ def endpoints(self):
 
     async def shutdown(self):
         await super(ChecksFinderContext, self).shutdown()
-        path = os.path.expandvars(r"%localappdata%/ChecksFinder")
-        for root, dirs, files in os.walk(path):
+        for root, dirs, files in os.walk(self.game_communication_path):
             for file in files:
                 if file.find("obtain") <= -1:
                     os.remove(root+"/"+file)
 
     def on_package(self, cmd: str, args: dict):
         if cmd in {"Connected"}:
-            if not os.path.exists(os.path.expandvars(r"%localappdata%/ChecksFinder")):
-                os.mkdir(os.path.expandvars(r"%localappdata%/ChecksFinder"))
+            if not os.path.exists(self.game_communication_path):
+                os.makedirs(self.game_communication_path)
```

Comment:
While it works nicely, I still see the issue of this line being jank on systems that don't have a ~/.wine yet. It would create one even though wine was never started.